spec/helpers/team_helper_spec.rb

```diff
@@ -20,4 +22,10 @@
       expect(can_edit_teams?(user)).to eq(true)
     end
   end
+
+  describe '#allowed_email?' do
+    it 'allows expected emails' do
+      expect(allowed_email?).to be false
+    end
+  end
```

Comment:
Could we have specs for all of the domain variants we're trying to support? Or at least an example where `allowed_email?` would be `true` ?

Reply:
My only hesitations was that having non `example.com` email addresses in specs kindof makes me uneasy BUT it should be fine here right?